src/diff.tsx

```diff
@@ -137,13 +143,24 @@ class MonacoDiffEditor extends React.Component<MonacoDiffEditorProps> {
   initMonaco() {
     const value =
       this.props.value != null ? this.props.value : this.props.defaultValue;
-    const { original, theme, options, overrideServices } = this.props;
+    const {
+      original,
+      theme,
+      options,
+      overrideServices,
+      className,
+    } = this.props;
     if (this.containerElement) {
       // Before initializing monaco editor
       this.editorWillMount();
+      const prependedOptions: monaco.editor.IStandaloneEditorConstructionOptions = {};
+      if (className) {
+        prependedOptions.extraEditorClassName = className;
+      }
       this.editor = monaco.editor.createDiffEditor(
         this.containerElement,
         {
+          ...prependedOptions,
```

Comment:
You can write

```suggestion
          ...(className ? {extraEditorClassName: className} : {}),
```

and simplify the code in all places.

Reply:
sure i can use a ternary.

Reply:
I think it makes the code clearer in this case (which cannot be said about ternaries in general ;-))